app/styles/app.scss

```diff
@@ -1448,3 +1448,41 @@ nav.branded-navbar {
     margin-left: 15px;
     text-decoration: underline;
 }
+
+.container.pretty-column {
+    .row {
+        padding-left: 20px;
+        .pretty-column-item {
+            padding-left: 20px;
+            &.pretty-column-top {
+                margin-bottom: 30px;
+            }
+        }
+    }
+}
+
+.seeAllSubjects {
+    margin-left: 15px;
+    text-decoration: underline;
+}
+
+.provider-logos {
+	display: flex;
```

Comment:
While you're in there, there are some tabs instead of spaces that are causing indent issues.